ocean_lib/models/erc721_token.py

```diff
@@ -220,8 +221,8 @@ def set_token_uri(
         )
 
     def create_datatoken(
-        self, erc20_data: CreateErc20Data, from_wallet: Wallet
-    ) -> ERC20Token:
```

Comment:
do you need the double typing? ERC20Enterprise is a subclass of ERC20Token.

Reply:
Forgot to delete it. Thanks.